packages/e2e-tests/helpers/source-panel.ts

```diff
@@ -4,7 +4,7 @@ import chalk from "chalk";
 import { openDevToolsTab } from ".";
 import { openPauseInformationPanel } from "./pause-information-panel";
 import { openSource, openSourceExplorerPanel } from "./source-explorer-panel";
-import { clearTextArea, debugPrint, delay, forEach, waitFor } from "./utils";
+import { clearTextArea, debugPrint, delay, forEach, waitFor, mapLocators } from "./utils";
```

Comment:
You don't actually use the new `mapLocators` function

Reply:
Yeah, I started to import it here, then realized that the logic that uses `Array.from()` in this file is subtly different - it's mapping over a subset of the locators, not all of them.  I can remove the import.

Reply:
On the other hand I could use that and slice later. That would work too.